keel-artifact/keel-artifact.gradle.kts

```diff
@@ -8,7 +8,6 @@ dependencies {
   implementation(project(":keel-igor"))
   implementation(project(":keel-clouddriver"))
   implementation("org.springframework:spring-context")
-  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core")
```

Comment:
Since I'm now exporting this via the `api` scope in `keel-core`, there's no need for individual modules to depend on it directly